docs/data/date-pickers/date-picker/date-picker.md

```diff
@@ -22,26 +30,33 @@ The date picker is rendered as a modal dialog on mobile, and a textbox with a po
 
 {{"demo": "BasicDatePicker.js"}}
 
-## Static mode
+## Uncontrolled vs. Controlled
 
-It's possible to render any date picker without the modal/popover and text field. This can be helpful when building custom popover/modal containers.
+The component can be controlled or uncontrolled
```

Comment:
Nitpick, but I'd suggest aligning the heading with the content, i.e.:
```diff
-Uncontrolled vs. Controlled
+Controlled vs. Uncontrolled
```
or 
```diff
- The component can be controlled or controlled
+ The component can be uncontrolled or controlled
```

Reply:
I copied the `TextField` wordings
https://mui.com/material-ui/react-text-field/#uncontrolled-vs-controlled

But I do agree it's not 100% coherent
I prefer your 2nd proposal (to match the order in the demo)